public/app/plugins/panel/table/migrations.ts

```diff
@@ -346,3 +348,43 @@ export const migrateHiddenFields = (panel: PanelModel<Partial<Options>>) => {
 
   return panel;
 };
+
+export const migrateFooterV2 = (panel: PanelModel<Options>) => {
```

Comment:
need unit tests for this method.

- [ ] confirm the existing footer options are deleted
- [ ] confirm the countRows case works as expected
- [ ] confirm that global footers work
- [ ] confirm that field-list footers work